CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [1.49](https://github.com/brave/brave-ios/milestone/106)
+
+Web3
+
+- Added Solana Swap integration into existing swap UI using Jupiter as the DEX aggregator ([#6847](https://github.com/brave/brave-ios/issues/6847))
+- Added message for Submitted/Completed state change after a transaction is submitted. ([#6960](https://github.com/brave/brave-ios/issues/6960))
```

Comment:
Can reword to `Added transaction submitted and transaction completed state in wallet panel`

Also linked PR instead of the issue (https://github.com/brave/brave-ios/issues/6861)